GOVERNANCE.rst

```diff
@@ -0,0 +1,183 @@
+
+We have the following guidelines:
+
+* We only consider published algorithms which have been shown to be competitive in comparative benchmarking studies or practically useful in applied projects. A technique that provides a clear-cut improvement (e.g. an enhanced data structure or a more efficient approximation technique) on a widely-used method will also be considered for inclusion.
+* From the algorithms or techniques that meet the above criteria, only those which fit well within the current framework and API of sktime are accepted. For extending current frameworks and API, see the [process for major changes]().
+* The contributor should support the importance of the proposed addition with research papers and/or implementations in other similar packages, demonstrate its usefulness via common use-cases/applications and corroborate performance improvements, if any, with benchmarks and/or plots. It is expected that the proposed algorithm should outperform the methods that are already implemented in sktime in at least some areas.
+* We strive to consolidate existing functionality when feasible. For example, when there are multiple techniques for the same purpose, we prefer to combine them into a single class and make case distinctions based on hyper-parameters.
+
+Note that your implementation need not be in sktime to be used together with
+sktime tools. You can implement your favorite algorithm in a sktime
+compatible way in one of `our companion repositories <https://github
+.com/sktime>`_ on GitHub. We will be happy to list it under `related
+software <https://github.com/alan-turing-institute/sktime/wiki/related
+-software>`_.
+
+If algorithms require major dependencies, we encourage to create a separate
+companion repository. For example, for deep learning techniques based on
+TensorFlow and Keras, we have `sktime-dl <https://github
+.com/sktime/sktime-dl>`_. For smaller dependencies which are limited to a few files, we encourage to use soft dependencies, which are only required for particular modules, but not for most of sktime's functionality and not for installing sktime.
+
+If significant issues are not fixed by the maintainer, and no other
+contributor volunteers to fix the algorithm within 3 months, we will
+deprecate and remove the algorithm from sktime eventually.
+
+Decision making
+---------------
+
+The purpose of this section is to formalize the decision-making process used by the sktime project. We clarify how decisions are made and who can make them.
+
+sktime's decision-making process is designed to take into account feedback from all community members and strives to find consensus, while avoiding deadlocks when no consensus can be found.
+
+Decisions about the future of the project are made through discussion with
+all members of the community. All discussion takes place on the project’s
```

Comment:
without a definition or at least indication of who is a "member of the community" this is ... pretty vague. All human beings? Everyone with a Github account? Only @fkiraly? :-)

Reply:
having said that, the roles below make things much clearer. Perhaps explain the high-level principles already here.